docs/release_notes/release_notes-0.16.0.rst

```diff
@@ -52,6 +52,7 @@ Key Features and Updates
   * FIX-#4993: Return `_default_to_pandas` in `df.attrs` (#4995)
   * FIX-#4597: Refactor Partition handling of func, args, kwargs (#4715)
   * FIX-#4996: Evaluate BenchmarkMode at each function call (#4997)
+  * FIX-#3764: Fix `df.loc` with scalar out of bounds appends to df (#3764)
```

Comment:
The number in parentheses should be the PR number, not the issue number (so #5046 in this case).